jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -443,8 +443,7 @@
         MKCLOUDTARGET=$mkcloudtarget
         [ $UPDATEBEFOREINSTALL == "true" ] && MKCLOUDTARGET='cleanup prepare setupadmin addupdaterepo instcrowbar setupcompute instcompute proposal testsetup'
 
-
-        if [ $WITHCROWBARREGISTER == "true" ] ; then
+        if [ $(uname -m) = s390x ]  || [ $WITHCROWBARREGISTER == "true" ] ; then
```

Comment:
how about
```bash
[ $(uname -m) = s390x ] && WITHCROWBARREGISTER=true
```

just in case we will be using that variable in other places later.